docs/reference/interface.md

```
@@ -179,6 +197,21 @@ _class mhyy.NotificationType_
 > **返回值:** [`NotificationType`](#notificationtype) - 枚举成员。
 
+## TimestampString {#timestampstring}
+
+> 时间戳格式的字符串。
+> 
+> str 的子类。
+
+_class mhyy.JSONString_
+(str)
+
+`to_time    `()
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements
null

(MD038, no-space-in-code)